tests/Microsoft.Bot.Builder.ApplicationInsights.Tests/TelemetryWaterfallTests.cs

```diff
@@ -225,26 +262,46 @@ public async Task EnsureCancelDialogCalled()
             Assert.IsTrue(saved_properties["WaterfallCancel_4"]["DialogId"] == "test");
             Assert.IsTrue(saved_properties["WaterfallCancel_4"].ContainsKey("StepName"));
             Assert.IsTrue(saved_properties["WaterfallCancel_4"].ContainsKey("InstanceId"));
+
             // Event contains "StepName"
             // Event naming on lambda's is "StepXofY"
             Assert.IsTrue(saved_properties["WaterfallCancel_4"]["StepName"] == "Step3of3");
             Assert.IsTrue(waterfallDialog.CancelDialogCalled);
             Assert.IsFalse(waterfallDialog.EndDialogCalled);
         }
 
+        private static WaterfallStep[] NewWaterfall()
```

Comment:
Since the intention of the PR is only to fix StyleCop issues, can we push this in a different PR to keep it consistent?

Thanks!

Reply:
Agreed! Moving it to a new PR